### PR TITLE
readme typo, silence warning, bump dependency.

### PR DESCRIPTION
**Pull request by matteoettam09**

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,4 +1 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
```
Comment (prevostc) [inline .husky/pre-commit:3]: why?

File: package.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "@graphprotocol/graph-cli": "^0.73.0",
+    "@graphprotocol/graph-cli": "^0.90.0",
```
Comment (prevostc) [inline package.json:41]: I just updated this in latest main so I don't have to review the changes on `yarn.lock`, can you rebase and drop your changes on both package.json and yank.lock plz 🙏

Comment (matteoettam09) [reply package.json]: Done

Comment (prevostc) [reply package.json]: `yarn.lock` changes still left

Comment (matteoettam09) [reply package.json]: Sorry, copy pasted now.

**Review by prevostc — APPROVED** [review]

**Pull request merged**